lib/ace/editor.js

```diff
@@ -70,7 +70,7 @@ var clipboard = require("./clipboard");
  *
  * @constructor
  **/
-var Editor = function(renderer, session) {
+var Editor = function(renderer, session, options) {
```

Comment:
not sure if it is worth to make session optional, to allow `new Editor(renderer, options)`